.github/workflows/nano_notebooks_tests.yml

```diff
@@ -0,0 +1,76 @@
+name: Nano Unit Tests for Notebooks
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'python/nano/**'
+      - '.github/workflows/nano_notebooks_tests.yml'
+
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  nano-notebooks-pytorch-test:
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-20.04"]
+        python-version: ["3.7"]
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade setuptools==58.0.4
+          python -m pip install --upgrade wheel
+
+      - name: Run notebooks trainer PyTorch unit tests
+        shell: bash
+        run: |
+          $CONDA/bin/conda create -n notebooks-trainer-pytorch -y python==3.7.10 setuptools=58.0.4
+          source $CONDA/bin/activate notebooks-trainer-pytorch
+          $CONDA/bin/conda info
+          bash python/nano/dev/build_and_install.sh linux default false pytorch
+          source bigdl-nano-init
+          pip install pytest nbmake
+          python -m pip install torch_ipex==1.9.0 -f https://software.intel.com/ipex-whl-stable
+          pip install lightning-bolts
+          bash python/nano/test/run-nano-notebooks-trainer-pytorch-tests.sh
+          source $CONDA/bin/deactivate
+          $CONDA/bin/conda remove -n notebooks-pytorch --all
+        env:
+          ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
+          
+      - name: Run notebooks inference Pytorch unit test
+        shell: bash
+        run: |
+          $CONDA/bin/conda create -n notebooks-infer-pytorch -y python==3.7.10 setuptools=58.0.4
+          source $CONDA/bin/activate notebooks-infer-pytorch
+          $CONDA/bin/conda info
+          bash python/nano/dev/build_and_install.sh linux default false pytorch
+          source bigdl-nano-init
+          pip install pytest nbmake
+          pip install lightning-bolts
+          pip install neural-compressor==1.11
+          pip install onnxruntime==1.10.0 onnx==1.11.0
+          pip install openvino-dev==2022.1.0
+          pip uninstall -y opencv-python
```

Comment:
which one introduced the `opencv-python`?

Reply:
openvino-dev